src/components/Card/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { ReactElement } from 'react';
+import { View, TouchableOpacity } from 'react-native';
+import Feather from '@expo/vector-icons/Feather';
+
+import { useNavigation } from '@react-navigation/native';
+
+import { Container, Photo, Name, Budget, ContentContainer } from './styles';
+
+interface CardProps {
+  name: string;
+  image: string;
```

Comment:
With this change the name of the props becomes clearer

```suggestion
  imageURL: string;
```